x-pack/plugins/observability_solution/apm/server/routes/alerts/register_apm_rule_types.ts

```diff
@@ -90,6 +90,7 @@ export const ApmRuleTypeAlertDefinition: IRuleTypeAlerts = {
   context: APM_RULE_TYPE_ALERT_CONTEXT,
   mappings: { fieldMap: apmRuleTypeAlertFieldMap },
   useLegacyAlerts: true,
+  shouldWrite: false,
```

Comment:
What does `shouldWrite` mean?

Reply:
We added this flag to prevent rules that were registered with both the rule registry and the alerting framework to have both write alerting docs. It will be removed this once we move all rules away from the rule registry

Reply:
We should set this to `true` here and remove the override in `x-pack/plugins/observability_solution/apm/server/routes/alerts/rule_types/transaction_duration/register_transaction_duration_rule_type.ts`